docs/ref/utils.txt

```diff
@@ -960,6 +960,11 @@ appropriate entities.
     The ``is_dst`` parameter has no effect when using non-``pytz`` timezone
     implementations.
 
+    .. deprecated:: 4.0
```

Comment:
Maybe we should move the entire paragraph about `pytz` under `deprecated` :thinking: Starting with ```When using ``pytz``... ```